.travis/run_autoformat.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8
+"""
+Wrapper script for doing auto-formatting in Travis.
```

Comment:
Stick it in a container, base it off: https://github.com/wellcometrust/dockerfiles/tree/master/build_tooling

Reply:
Yeah, this is just a POC/WIP.